clean.cmd

```diff
@@ -11,8 +11,8 @@ if /I [%NO_DASHES_ARG:-=%] == [h] goto Usage
 tasklist /fi "imagename eq VBCSCompiler.exe" |find ":" > nul
 :: Compiler is running if errorlevel == 1
 if errorlevel 1 (
-	echo Stop VBCSCompiler.exe execution.
-	for /f "tokens=2 delims=," %%F in ('tasklist /nh /fi "imagename eq VBCSCompiler.exe" /fo csv') do taskkill /f /PID %%~F
+  echo Stop VBCSCompiler.exe execution.
```

Comment:
Should this script / functionality be a part of eng/common such that https://github.com/dotnet/arcade/blob/master/eng/common/init-tools.ps1#L193 can just share the functionality?

Reply:
I think so that is why I filed https://github.com/dotnet/arcade/issues/906 but I didn't want to tackle that issue just yet.